aws/resource_aws_transfer_server_test.go

```diff
@@ -111,6 +111,51 @@ func TestAccAWSTransferServer_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSTransferServer_Vpc(t *testing.T) {
+	var conf transfer.DescribedServer
+
```

Comment:
add `resourceName := "aws_transfer_server.test"` and replace all `"aws_transfer_server.test"` occurrences